exercises/armstrong-numbers/lib/example.dart

```diff
@@ -0,0 +1,17 @@
+import 'dart:math' show pow;
+
+class ArmstrongNumbers {
+  bool isArmstrongNumber(final int number) {
+    final String numberAsString = number.toString();
+    final int numOfDigits = numberAsString.length;
+    int sum = 0;
+
+    for(int count = 0; count < numOfDigits; count++) {
```

Comment:
Suggestion : introduce `foreach`

Reply:
The example solution shouldn't be readied seen by the user. So it wouldn't matter whether it uses foreach or not.

Though, we could add a suggestion to the README. Maybe a link to an example on dartlang.org?

Reply:
Should we add a discoverable way for students to checkout example.dart?

Reply:
Reference links to dartlang website for syntax/language specific hints is a good idea.